tests/rustdoc-json/reexport/issue-112852-dangling-trait-impl-id.rs

```diff
@@ -0,0 +1,12 @@
+#![feature(no_core)]
```

Comment:
NIT: this should be in the `impls` or `traits` folder, as there are no reexports

Reply:
I don't know why I put it there... Moving the location.